src/main/java/tech/allegro/wildsnake/showcase/service/ShowcaseService.java

```diff
@@ -26,7 +26,7 @@ public ShowcaseService(ProductRepository productRepository) {
 
     private List<ShowcaseItem> prepareShowcaseItems(List<Product> productList) {
         return productList.stream().map(
-                product -> new ShowcaseItem(product.getName(), product.getImageUrl())
+                product -> new ShowcaseItem(product.getName(), product.getImageUrl(), product.getPrice())
```

Comment:
aż miłobyłoby mieć klasę ShowcaseItemBuilder, która to miałaby metodę potrafiącą zbudować ten obiekt na podstawie obiektu typu Product